frontend/src/App.tsx

```diff
@@ -2,33 +2,49 @@ import React from 'react';
 import SimpleGet from './components/SimpleGetRequestTemplate';
```

Comment:
Remove this library if it's not used to fix lint

Reply:
fixed just now